src/components/SimpleSelect.js

```diff
@@ -82,17 +74,29 @@ class SimpleSelect extends React.Component {
           {this.props.children ?
             this.props.children :
             (this.props.items.map((item, i) => {
+              const { icon, isSelected, onClick, text, ...rest } = item;
+
               return (
                 <Option
+                  isSelected={isSelected}
```

Comment:
Is this needed if you're just passing it straight down? Seems like it could pass along with `...rest`.

Reply:
Technically it doesn't have to be but I left it this way because it seemed more like a public api prop where as the over writing I need to do the rest I was going to leave undocumented.